llama_stack/providers/remote/telemetry/opentelemetry/config.py

```diff
@@ -8,5 +8,5 @@
 
 
 class OpenTelemetryConfig(BaseModel):
```

Comment:
can you add a `sample_run_config()` here while you are at it. it will be useful some day.

Reply:
done.